apps/server/src/modules/lesson/controller/dto/lesson-content.response.ts

```diff
@@ -9,7 +9,7 @@ import {
 	ComponentType,
 } from '@shared/domain/entity/lesson.entity';
 import { EntityId } from '@shared/domain/types';
-
+import { LessonResources } from './lesson-resources.model';
 // eslint problem will be solved in EW-1090
```

Comment:
Please add the missing modifier everywhere to make the linter warnings disappear.